model/fn-execution/src/main/proto/beam_fn_api.proto

```diff
@@ -296,6 +296,15 @@ message ProcessBundleRequest {
   // (Optional) A list of cache tokens that can be used by an SDK to reuse
   // cached data returned by the State API across multiple bundles.
   repeated CacheToken cache_tokens = 2;
+
+  // (optional) Elements to be processed with the bundle. Either all or
+  // none of the bundle elements should be included in the ProcessBundleRequest.
+  // This elements embedding is to achieve better Fn API efficiency for bundles
+  // that contain only small amount of data and are cheap to be processed on the
+  // SDK harness side. This field can be set only if the SDK declares that it
+  // supports the CONTROL_REQUEST_ELEMENTS_EMBEDDING capability. See more at
```

Comment:
```suggestion
  // supports the beam:protocol:control_request_elements_embedding:v1 capability. See more at
```

Note that the ENUM can change but its contents of string can't so best to use it within the comment.

Reply:
Done.